tests/test_transforms/test_missing_values/test_impute_transform.py

```diff
@@ -251,6 +257,51 @@ def test_missing_values_seasonal(ts_to_fill, window: int, seasonality: int, expe
     np.testing.assert_array_equal(result, expected)
 
 
+@pytest.mark.filterwarnings("ignore: Mean of empty slice")
+@pytest.mark.parametrize(
+    "window, seasonality, expected",
+    [
+        (
+            1,
+            3,
+            np.array(
+                [[np.NaN, 0, np.NaN, 2, 3, 4, 5, 3, 4, 8, 9], [np.NaN, 10, np.NaN, 30, 40, 50, 60, 40, 50, 90, 100]]
+            ).T,
+        ),
+        (
+            3,
+            1,
+            np.array(
+                [[np.NaN, 0, 0, 2, 3, 4, 5, 4, 9 / 2, 8, 9], [np.NaN, 10, 10, 30, 40, 50, 60, 50, 110 / 2, 90, 100]]
```

Comment:
Is this the only test case this is different with `test_missing_values_seasonal`?

Reply:
Yes)

Reply:
Do you think we need more?